doc/toggleterm.txt

```diff
@@ -477,13 +477,18 @@ using the current count e.g. `:5ToggleTerm<CR>`
     local lazygit = Terminal:new({ cmd = "lazygit", count = 5 })
 <
 
-You can also set a custom layout for a terminal.
+You can also set several options for the custom terminal including layout,
+opening and closing functions, and the newline charcter used when sending text
+to the terminal. The `newline_chr` defaults to "\r" on powershell, "\r\n" on
+cmd.exe, and "\n" otherwise.
 
 >lua
     local lazygit = Terminal:new({
       cmd = "lazygit",
       dir = "git_dir",
       direction = "float",
+      -- Try changing if you get too many/few blank lines when sending to terminal
```

Comment:
I don't think we need a special addition for this especially given the explainer above

Reply:
removed